cloudant-sync-datastore-core/src/main/java/com/cloudant/mazha/CouchClient.java

```diff
@@ -541,7 +542,7 @@ private void closeQuietly(InputStream is) {
         }
     }
 
-    InputStream bulkInputStream(List<?> objects) {
+    private InputStream bulkPostInputStream(List<?> objects) {
```

Comment:
This is slightly misleading from the aspect that a `_bulk_get` request is also a `POST`, but maybe that is just because I know that endpoint detail. Maybe `bulkInsert` & `bulkRetrieve` make the intent more clear without the ambiguity of the HTTP request type?